style-sheets/primevue-components-overrides.scss

```diff
@@ -271,5 +295,57 @@ label {
 }
 
 /* ------------------------------------ primevue card--------------------------------- */
+.#{variables.$primevue-prefix}-card {
+    box-shadow: none;
+    border: 1px solid colors.$gray-30;
+    padding: 32px;
+}
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-body {
+    padding: 32px;
+}
+
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-content {
+    padding: 0;
+    margin: 0;
+}
+
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-title {
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 400;
+    margin-bottom: 0;
+}
 
+.#{variables.$primevue-prefix}-card .#{variables.$primevue-prefix}-card-subtitle {
+    color: var(--light-theme-text-text-secondary, #606062);
```

Comment:
let's try to find if this color is in the design token folder

Reply:
Thanks Catherine, for the comments. I have updated this.